alternate/Tenkuu Danzai Skelter Heaven (alternate).ass

```diff
@@ -26,213 +28,220 @@ Style: OPTL,Monotype Corsiva,22,&H00F8F091,&H000000FF,&H00000000,&H7F000000,-1,0
-Dialogue: 0,0:08:31.62,0:08:33.36,Default,,0,0,0,,You're a loose cannon.
+Dialogue: 0,0:07:25.82,0:07:26.79,Default,,0,0,0,,You're fast.{I love this}
+Dialogue: 0,0:07:26.91,0:07:28.25,Default,,0,0,0,,Attack the core.
+Dialogue: 0,0:07:29.60,0:07:30.60,Default,,0,0,0,,Yes.
+Dialogue: 0,0:07:30.66,0:07:31.43,Default,,0,0,0,,{\an8}TL:note Taichou means commanding officer
+Dialogue: 0,0:07:30.66,0:07:31.43,Default,,0,0,0,,Taichou!
+Dialogue: 0,0:07:31.51,0:07:34.33,Default,,0,0,0,,{\an8}TL:Note Japan Self-Defense Forces is the JSDF
+Dialogue: 0,0:07:31.51,0:07:34.33,Default,,0,0,0,,The JSDF is attacking the target.
+Dialogue: 0,0:07:42.50,0:07:46.16,Default,TLC,0,0,0,,The targets energy reading has gotten big enough.
+Dialogue: 0,0:07:57.68,0:07:58.42,Default,TM,0,0,0,,Oh...{Pls make timing badder - Lann}
+Dialogue: 0,0:08:00.03,0:08:02.06,Default,,0,0,0,,Captain, I am not conent.
+Dialogue: 0,0:08:02.13,0:08:03.80,Default,,0,0,0,,Why am I second?
+Dialogue: 0,0:08:03.87,0:08:05.65,Default,,0,0,0,,My commands cause you problems?
+Dialogue: 0,0:08:06.59,0:08:07.69,Default,,0,0,0,,Yes.
+Dialogue: 0,0:08:08.03,0:08:10.05,Default,,0,0,0,,Rin, please go ahead.
+Dialogue: 0,0:08:16.96,0:08:19.33,Default,,0,0,0,,Is it okay? To let Rin go?
+Dialogue: 0,0:08:19.39,0:08:20.58,Default,,0,0,0,,What do you mean?
+Dialogue: 0,0:08:20.64,0:08:22.98,Default,,0,0,0,,I don't let my feelings interfere with work!
+Dialogue: 0,0:08:23.82,0:08:26.08,Default,,0,0,0,,Then make me fast!
+Dialogue: 0,0:08:26.15,0:08:28.28,Default,TLC,0,0,0,,I smell better than Rin.{Too far? (全て匂い...) - Lann I like it}
+Dialogue: 0,0:08:28.37,0:08:29.35,Default,,0,0,0,,I can do it.
+Dialogue: 0,0:08:29.55,0:08:31.57,Default,,0,0,0,,You are too much on your own authority.
+Dialogue: 0,0:08:31.62,0:08:33.36,Default,,0,0,0,,It will be a hassle.
 Dialogue: 0,0:08:37.20,0:08:38.70,Default,,0,0,0,,So that's it.
-Dialogue: 0,0:08:38.80,0:08:41.53,Default,,0,0,0,,I'm inferior to Rin, am I?
+Dialogue: 0,0:08:38.80,0:08:41.53,Default,,0,0,0,,I'm worse than Rin, am I?
 Dialogue: 0,0:08:42.68,0:08:44.04,Default,,0,0,0,,Misaki...
 Dialogue: 0,0:08:44.86,0:08:46.08,Default,,0,0,0,,I'll show him.
-Dialogue: 0,0:08:47.18,0:08:49.08,Default,,0,0,0,,I'll {\i1}prove{\i0} I'm superior to Rin.
-Dialogue: 0,0:08:53.69,0:08:58.45,Default,TLC,0,0,0,,Ensure you penetrate the target directly with the probe.
-Dialogue: 0,0:08:58.51,0:09:00.07,Default,,0,0,0,,With this improvised weapon?
-Dialogue: 0,0:09:00.96,0:09:03.49,Default,,0,0,0,,01 Skelter Wave, copy that.
-Dialogue: 0,0:09:21.44,0:09:23.94,Default,,0,0,0,,Skelter Wave, preparing for launch.
+Dialogue: 0,0:08:47.18,0:08:49.08,Default,,0,0,0,,I'll show I'm better then Rin.
+Dialogue: 0,0:08:53.69,0:08:58.45,Default,TLC,0,0,0,,Ensure you use the panzerfaust to penetrate the target. {this hurts to hear}
+Dialogue: 0,0:08:58.51,0:09:00.07,Default,,0,0,0,,With this quickly made weapon?
+Dialogue: 0,0:09:00.96,0:09:03.49,Default,,0,0,0,,01 skelter wave, roger.
```

Comment:
Thoughts on adding dashes in between the numbers to make it clear that they should be said separately?

```suggestion
Dialogue: 0,0:09:00.96,0:09:03.49,Default,,0,0,0,,0-1 skelter wave, roger.
```